kweb/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM kopano/kwebd:0.4.2
+RUN apk add --no-cache libcap && \
+    setcap 'cap_net_bind_service=+ep' /usr/local/bin/kwebd
```

Comment:
It would be better if we would only listen on two ports (like 8080 and 8443) which automatically does not require that special cap.